frappe/core/doctype/file/file.py

```diff
@@ -41,7 +41,6 @@
 from frappe.utils.image import optimize_image, strip_exif_data
 
 if TYPE_CHECKING:
-	from PIL.ImageFile import ImageFile
```

Comment:
This is being used: 

https://github.com/frappe/frappe/blob/c8a8c42f79a7510f88774b68a7fd8e11bd8c6f2c/frappe/core/doctype/file/file.py#L723 


Looks like pycln doesn't check for strigified annotations 😬

Reply:
`ImageFile` is already imported at the top?

Reply:
ah lol ok 👍